fcrepo-http-api/src/main/resources/views/common.js

```diff
@@ -80,7 +80,13 @@
       };
       reader.readAsArrayBuffer(update_file);
     } else {
-      fn(method, url, headers, null);
+    	const turtle = document.getElementById('turtle_payload')
```

Comment:
Minor point: you should have semicolons at the end of statements here. They aren't technically needed, but we should at least be consistent with the javascript. Also check the indentation to make it consistent -- most indentation is two spaces, not three or four as you have here.